src/methods.js

```diff
@@ -24,7 +24,9 @@ var methods = {};
             verdictLevel,
             score;
 
-        if (options === undefined) { return; }
+        if (typeof(options) === "undefined") {
```

Comment:
Crockford says the `typeof` form is slower and unnecessary. I personally don't think this is a big deal, but I like to keep the whole project passing the linter checks.

I've changed linter and in my new projects I use ESLint, but this one was set up with JSLint, and I am too lazy to change it.